requirements.txt

```diff
@@ -24,3 +24,4 @@ click >= 7
 click-option-group >= 0.5, < 0.6
 openpyxl
 fsspec
+pydantic>=1.4
```

Comment:
In the effort of moving towards typing and making the code more robust, the use of pydantic seems to be relevant.  But please concert before introducing new dependencies to the requirements.